### ISSUE TITLE
RequestLogger: logResponseBody hangs if the response is large (200KB)

### ISSUE THREAD

**Issue by eug48** [DevExpress/testcafe#2725]
### Are you requesting a feature or reporting a bug?

Bug

### What is the current behavior?

Setting logResponseBody to true seems to cause large responses to not be received by the browser.

### What is the expected behavior?

logResponseBody should work correctly

### How would you reproduce the current behavior (if this is a bug)?

Modify the log-options.js test case to use a large file (200KB) instead of index.html. I have not tried as unfortunately I cannot get these tests to run (syntax error on import statement).

#### Provide the test code and the tested page URL (if applicable)

Tested page URL:

Test code

```js
const gqlLogger = RequestLogger(/graphql/, {
  logResponseBody: true,
});
```

### Specify your

* operating system: Windows 10 1803
* testcafe version: 0.21.1
* node.js version: 10.3.0

**Comment by miherlosev** [comment]
Hi @eug48 

Thank you for your report. I've reproduced the problem.
Also, I've attached a [simple example](https://github.com/DevExpress/testcafe/files/2286199/temp.zip) that demonstrates the problem.

**Comment by miherlosev** [comment]
I'm working on this

**Comment by miherlosev** [comment]
The fix was published in `testcafe@0.22.1-alpha.3`
